libby/layout/Table.lib.js

```diff
@@ -90,15 +95,15 @@ describe('Table', () => {
           </Table.Row>
         </thead>
         <tbody>
-          <Table.Row p="800">
```

Comment:
Removed this because padding doesn't work on `tr`. I don't think it ever did, doesn't look like we were overwriting padding context in the row